tests/util/test_unit_conversion.py

```diff
@@ -32,59 +35,28 @@
 )
 
 INVALID_SYMBOL = "bob"
+_CONVERTERS: dict[type[BaseUnitConverter], list[str]] = {
+    obj: sorted(obj.VALID_UNITS)
+    for _, obj in inspect.getmembers(unit_conversion)
```

Comment:
It's nice that we don't need to update this container, but it's not very readable what the contents is besides the type.

Reply:
Yeah I think it might be better to add a test to ensure all converters are tested instead.

Reply:
@MartinHjelmare I've opened #86271 as an alternative.
I think it is better to keep the generated list just for comparing against the manual list and not for actually running the tests.

Reply:
Sounds good!